index.html

```diff
@@ -39,22 +39,276 @@ <h1 class="main-text font-poppins">
           <ul>
             <li>
               <a href="https://twitter.com/_JavierAybar">
-                <img class="social-media-icon" src="images/Twitter-icon.svg" alt="Twitter" />
+                <img
+                  class="social-media-icon"
+                  src="images/Twitter-icon.svg"
+                  alt="Twitter"
+                />
               </a>
             </li>
             <li>
               <a href="https://www.linkedin.com/in/javier-aybar-932376274/">
-                <img class="social-media-icon" src="images/Linkedin icon.svg" alt="Linkedin" />
+                <img
+                  class="social-media-icon"
+                  src="images/Linkedin icon.svg"
+                  alt="Linkedin"
+                />
               </a>
             </li>
             <li>
               <a href="https://github.com/JavierAybar">
-                <img class="social-media-icon" src="images/Github-icon.svg" alt="GitHub" />
+                <img
+                  class="social-media-icon"
+                  src="images/Github-icon.svg"
+                  alt="GitHub"
+                />
               </a>
             </li>
           </ul>
         </section>
       </section>
     </main>
+    <!--Works-->
+    <section class="cards-container-flex">
+      <section class="cards">
+        <article class="card">
+          <figure>
+            <img
+              class="img-card"
+              src="images/SnapshootPortfolio(1).svg"
+              alt=""
+            />
+          </figure>
+
+          <section class="card-description">
+            <header class="header-card">
```

Comment:
the header tag is not the right tag to use as this is the definition : The <header> element represents a container for introductory content or a set of navigational links.
you can use a div to wrap these items